src/Components/AddForm.js

```diff
@@ -44,22 +47,18 @@ class AddForm extends Component {
     if (this.subject.value === '') {
       alert('Subject can\'t be empty');
     } else {
-      this.setState({
-        addedItem: {
-          id: shortid.generate(),
-          subject: this.subject.value,
-          category: this.category.value,
-          assignee: this.assignee.value,
-          priority: this.priority.value,
-          status: 'Open',
-          update: false,
-        },
-      }, function sendProps() {
-        const { addedItem } = this.state;
-        const { addItem } = this.props;
-        addItem(addedItem);
-        this.formReset();
-      });
+      const { addItem } = this.props;
+      const item = {
+        id: shortid.generate(),
+        subject: this.subject.value,
+        category: this.category.value,
+        assignee: this.assignee.value,
+        priority: this.priority.value,
```

Comment:
> When you're using React, avoid to manipulate the real DOM.
Try to put those input data into `state`
```js
// state
this.state = {value: ''};

// onChange callback
handleChange(event) {
    this.setState({value: event.target.value});
  }

// element
<input type="text" value={this.state.value} onChange={this.handleChange} />
```

Ref: https://reactjs.org/docs/forms.html